mkdocs.yml

```diff
@@ -0,0 +1,51 @@
+site_name: OpenStack Data Plane Adoption
+site_description: OpenStack Data Plane Adoption
+site_author: OpenStack Team
+site_url: https://openstack-k8s-operators.github.io/data-plane-adoption
+repo_name: openstack-k8s-operators/data-plane-adoption
+repo_url: https://github.com/openstack-k8s-operators/data-plane-adoption
+theme:
+  features:
+    - search.suggest
+    - search.highlight
+    - search.share
+  name: material
+  logo: images/openstack-logo.png
+  palette:
+    - scheme: default
+      media: "(prefers-color-scheme: light)"
+      primary: teal
+      toggle:
+        icon: material/toggle-switch-off-outline
+        name: Switch to dark mode
+    - scheme: slate
+      media: "(prefers-color-scheme: dark)"
+      primary: teal
+      toggle:
+        icon: material/toggle-switch
+        name: Switch to light mode
+nav:
+  - README.md
+  - OpenStack:
+    - backend_services_deployment.md
+    - mariadb_copy.md
+    - keystone_adoption.md
+    - glance_adoption.md
+    - other_services_adoption.md
+  - Ceph:
+    - ceph.md
+  - troubleshooting.md
```

Comment:
@jistr FYI this is how we define the sidebar

Reply:
This is actually nice and really useful, I'm going to have the RGW adoption section under `Ceph`, so it's nice I can define I list of documents to provide a better navigation!